_src/assets/js/main.js

```diff
@@ -86,4 +86,31 @@ function applyPalette(e) {
 // En este caso, no se porque... no me funciona con 'click' y si con 'change'
 for (const c of colors) {
   c.addEventListener('change', applyPalette);
-}
\ No newline at end of file
+}
+
+// Añadir imagen
+
+const fr = new FileReader();
+const uploadBtn = document.querySelector('.btn__add-img');
+const fileField = document.querySelector('#btn__add-img');
+const profileImage = document.querySelector('.card__img');
+const divPreviewImage = document.querySelector('.square__img');
+
+function getImage(e){
+  const myFile = e.currentTarget.files[0];
+  fr.addEventListener('load', writeImage);
+  fr.readAsDataURL(myFile);
+}
+
+function writeImage() {
+  console.log('Hola');
```

Comment:
Do we really need a hi!! message from the console?

Reply:
We forgot about it haha
I've already deleted it.
Thanks.